apps/builder/app/shared/copy-paste/plugin-webflow/plugin-webflow.test.tsx

```diff
@@ -72,7 +72,7 @@ const toCss = (fragment: WebstudioFragment) => {
   }
   const ruleByStyleSourceId = new Map<string, NestingRule>();
   for (const styleDecl of fragment.styleSources) {
-    const name = styleDecl.type === "token" ? styleDecl.name : "Local";
+    const name = "name" in styleDecl ? styleDecl.name : "Local";
```

Comment:
why?

Reply:
dunno, felt like in the moment its more readable aka "if there is name, uuse the name, otherwise local", can revert

Reply:
type makes more sense. in is always vague check